themes/default/content/blog/slack-integration/index.md

```diff
@@ -0,0 +1,399 @@
+
+## Filtering Notification Events
+
+Event filtering allows you to select which events should be delivered to each webhook. You may want to receive
+all events, or filter to specific events (only failures, only deployment events, etc.). The following table describes the various event filters available and the context in which they are relevant.
+
+| Filter                 | Event Kind      | Webhook Type               | Triggered                         |
+|------------------------|-----------------|----------------------------|-----------------------------------|
+| `stack_created`        | `stack`         | Organization webhooks only | When a stack is created.          |
+| `stack_deleted`        | `stack`         | Organization webhooks only | When a stack is deleted.          |
+| `preview_succeeded`    | `stack_preview` | Both                       | When a stack `preview` succeeds.  |
+| `preview_failed`       | `stack_preview` | Both                       | When a stack `preview` fails.     |
+| `update_succeeded`     | `stack_update`  | Both                       | When a stack `update` succeeds.   |
+| `update_failed`        | `stack_update`  | Both                       | When a stack `update` fails.      |
+| `destroy_succeeded`    | `stack_update`  | Both                       | When a stack `destroy` succeeds.  |
+| `destroy_failed`       | `stack_update`  | Both                       | When a stack `destroy` fails.     |
+| `refresh_succeeded`    | `stack_update`  | Both                       | When a stack `refresh` succeeds.  |
+| `refresh failed`       | `stack_update`  | Both                       | When a stack `refresh` fails.     |
+| `deployment_queued`    | `deployment`    | Both                       | When a deployment is queued.      |
+| `deployment_started`   | `deployment`    | Both                       | When a deployment starts running. |
+| `deployment_succeeded` | `deployment`    | Both                       | When a deployment succeeds.       |
+| `deployment_failed`    | `deployment`    | Both                       | When a deployment fails.          |
+
+## Wrapping up
+
+We hope you and your team can streamline deployment related notifications to where you spend your time. Our webhooks improvements enable ChatOps workflows and more visibility into your infrastructure. As always, please let us know if you have feedback on the feature by opening an issue in the Pulumi Cloud requests repository or if you have other features you would like to see in Pulumi Cloud.
```

Comment:
Same "streamline" comment as earlier, and also feels a reiteration of the opening line. How about something like this maybe?

```suggestion
These new features are all about getting you the information you need wherever you are, and we can't wait to see how you use them. Our webhooks improvements enable ChatOps workflows and more visibility into your infrastructure. As always, please let us know if you have feedback on the feature by opening an issue in the Pulumi Cloud requests repository or if you have other features you would like to see in Pulumi Cloud.
```